src/libcephsqlite.h

```diff
@@ -0,0 +1,19 @@
+#ifndef LIBCEPHSQLITE_H
+#define LIBCEPHSQLITE_H
+
+#include "rados/librados.hpp"
+
+struct sqlite3;
+
+extern "C"
+sqlite3 *ceph_sqlite3_open(
+  librados::Rados &cluster,
+  const char *dbname,           /* eg. "kvstore" instead of "kvstore.db" */
+  const char *rados_namespace,
+  int ceph_pool_id,
+  int stripe_count,
```

Comment:
I see why you have these options but I think `stripe_count`/`obj_size` should be configured in a deferred fashion.